src/libraries/System.Net.Security/tests/UnitTests/NTAuthenticationTests.cs

```diff
@@ -16,6 +18,7 @@ public class NTAuthenticationTests
 
         private static NetworkCredential s_testCredentialRight = new NetworkCredential("rightusername", "rightpassword");
         private static NetworkCredential s_testCredentialWrong = new NetworkCredential("rightusername", "wrongpassword");
+        private static byte[] s_Hello = new byte[] { (byte)'H', (byte)'e', (byte)'l', (byte)'l', (byte)'o' };
```

Comment:
Are the new UTF-8 literals in C# yet? Does runtime use new enough compiler?

Reply:
[Yup](https://github.com/dotnet/runtime/pull/68334)

Reply:
Thanks 👍 I will update the code to use it.